lib/routes/security_pin/remote_server_auth.dart

```diff
@@ -370,8 +409,25 @@ class RemoteServerAuthPageState extends State<RemoteServerAuthPage> {
       return DiscoverResult.INVALID_AUTH;
     } on RemoteServerNotFoundException {
       return DiscoverResult.INVALID_URL;
+    } on MethodNotFoundException {
+      return DiscoverResult.METHOD_NOT_FOUND;
     }
 
+    // Since we are restoring we cannot be sure whenever the account has any
+    // backups.
+    if (widget.restore) {
```

Comment:
This code doesn't belong here. We have a flow for restore and no available backups the restore won't work anyway without this change.
If we want to add an additional error message we can do that as part of the restore flow.

Reply:
I have removed this logic as agreed.